client/js/components/GameView/Card.vue

```diff
@@ -78,6 +78,8 @@ export default {
           return 'Hearts';
         case 3:
           return 'Spades';
+        case 'back':
```

Comment:
Back also isn't a suit name so this doesn't belong here. This logic should be forked off and a decision made elsewhere before it get here. If you make the above changes, you could fast fail here with a number check and return `'Invalid Suit Error'` early or just remove the back entry here.

You'd probably want another computed prop for `isBack` that could use a check like `!suit || !rank` or whatever to return `true`, and use that instead.